app/templates/account/prepaid-view.pug

```diff
@@ -50,42 +59,43 @@ block content
                   .col-xs-6
                     span
                       button#redeem-code-btn.btn.btn-success.btn-block(data-i18n="account_prepaid.apply_account")
-      .row
-        .col-md-12
-          #codes-panel.panel.panel-default
-            .panel-heading
-              .panel-title
-                a(data-toggle="collapse" href="#codeslist")
-                  span(data-i18n="account_prepaid.your_codes")
-            .panel-collapse.collapse.in#codeslist
-              .panel-body
-                if view.codes && view.codes.length
-                  table.table.table-striped
-                    tr
-                      th(scope="col")
-                        span.spr(data-i18n="[title]account_prepaid.copy_link;general.code", title="You can copy the code's link and send it to someone.")
-                          span.glyphicon.glyphicon-question-sign(aria-hidden="true")
-                      th(scope="col" data-i18n="account_prepaid.months")
-                      th(scope="col") Remaining Users
-                      th(scope="col") Total Users
-                      th(scope="col" data-i18n="user.status")
-                    for code in view.codes.models
-                      if code.get('type') === 'terminal_subscription'
-                        - var owner = (code.get('creator') == me.id ? true : false)
-                        - var properties = code.get('properties')
-                        - var redeemers = code.get('redeemers')
-                        - var redeemed = redeemers ? redeemers.length : 0
-                        tr
-                          td
-                            a(href='/account/prepaid?_ppc=' + code.get('code'))= code.get('code')
-                          td= properties.months || '-'
-                          if owner
-                            td= code.get('maxRedeemers') - redeemed
-                            td= code.get('maxRedeemers')
-                            td(data-i18n="account.purchased")
-                          else
-                            td -
-                            td -
-                            td(data-i18n="account_prepaid.redeemed")
-                else
-                  p(data-i18n="account_prepaid.no_codes")
+      if !features.chinaHome && false
```

Comment:
assuming u forgot to remove?

Reply:
nope. i just want to remove this part. but not sure if nick still want it. so adding a `false` to make is fake-removed :joy:

Reply:
haha ok remove it or remove false after confirming with Nick 😅

Reply:
but yeah we can remove these part and in case we still need that we can revert them from git history